data/json/mapgen/homeimprovement_superstore_new.json

```diff
@@ -314,6 +314,7 @@
       "s": "f_sink",
       "b": "f_bathtub",
       "(": "f_shower",
+      "é": "f_freezer",
```

Comment:
You have to give it a terrain entry as well.   Otherwise the game thinks you are loading furniture over a void.